### DISCUSSION TITLE
Deploy via github actions to firebase - repo in organization

### DISCUSSION THREAD

**Discussion by ghost** [community/community#27470] [Actions] [answered]
Hi.

I try to make a CI/CD setup for my deployment to firebase.
My repo is in a organization and my account have admin rights to the repo.
But whenever i try to type the Github repo path (format: user/repo) as firebase CLI requires. it can't proceed - no errors, the terminal is just flashing the line. For me it seems like it maybe can't access a repo in organization?
![image](https://user-images.githubusercontent.com/48101800/181498918-86c0fcc8-1e6c-4b75-8df1-888efd81f47d.png)

Can some provide me with ino on how to proceed?

**Comment by andrea-acampora** [comment, marked as answer]
Try this Github Action: [action-link](https://github.com/marketplace/actions/deploy-to-firebase-hosting).
It seems that it deploys the current state of your GitHub repo to your live channel when a PR is merged.

**Reply by ghost** [reply]
Hi @andrea-acampora

It's exactly what i tried to do - only difference is i want to make Firebase CLI do the yml work (By running firebase init hosting:github command) and it's in this progress it won't connect to the repo.

Could it be something that Firebase CLI can't connect to a organizational repo, because they require in format user/repo ? and can't convert user/organization ?

**Reply by andrea-acampora** [reply]
@Octavius-Palmqvist Yes, the problem is that the repo is inside an organization. You can check the issue firebase/firebase-tools#2763 .
Btw there's a workaround: first you have to grant Firebase CLI 3rd party github app in [github-app](https://github.com/settings/applications) and then do `firebase init hosting:github` and enter it as organization/repo-name.

**Reply by ghost** [reply]
Sweet jesus, it works :) Thx alot @andrea-acampora !

**Reply by AirZED** [reply]
Wow, this worked

**Reply by blackright** [reply]
Wow, this worked.

**Comment by Pavubrar** [comment]
Hi @andrea-acampora I have the same issue before, I tried the way told here, but now I got another error, 
Error: HTTP Error: 403, Policy update access denied.

 even my organization gives me access as well.

**Reply by tdebooij** [reply]
I have the exact same issue, firebase-cli has full access for the organisation, and I am admin of the repo. Still getting `HTTP Error 403, Policy update access denied`. I've tried downgrading firebase-tools but that does not help either.
